app/Mage.php

```diff
@@ -1002,6 +996,7 @@ public static function printException(Throwable $e, $extra = '')
 
             require_once(self::getBaseDir() . DS . 'errors' . DS . 'report.php');
         }
+        self::dispatchEvent('mage_print_exception_after', ['exception' => $e, 'extra' => $extra]);
```

Comment:
is this needed? since probably the mage_print_exception_before will end with a die...

and the normal error report.php also dies, there's probably no way anybody could reach the mage_print_exception_after

Reply:
I was thinking about possible additional error tracking integrations after print / save reports
to me report.php call Error_Processor that render the template only after reach die() in printException(), correct?

Reply:
my bad, you're right, the die is AFTER this code so my point is invalid